src/layouts/Sidebar.tsx

```diff
@@ -661,12 +662,14 @@ const Sidebar = () => {
                 width="100%"
               >
                 <Icon name={Icons.EYE} />
-                <BetaVersionToggle
-                  checked={isNewCreateEnabled}
-                  onChange={() => {
-                    setIsNewCreateEnabled((prev) => !prev);
-                  }}
-                />
+                {isClient && (
+                  <BetaVersionToggle
```

Comment:
I wonder if we still need the BetaVersionToggle.
Maybe this component can even be removed. 
If this is the case we might not need the isClient check